app/Tasks/Status.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace App\Tasks;
+
+use App\Builder\Docker;
+use App\Builder\DockerCompose;
+use App\Builder\Escaped;
+
+class Status extends Task
+{
+    /** @var bool $quite */
+    protected $quite = false;
+
+    public function run(...$args): int
+    {
+        $start = Start::make($this->command);
+
+        $chain = [
+            [$start, 'handleNetwork'],
```

Comment:
why do we have to handle network here? does it fails to check if the network isn't already created? i don't think checking status justifies creating networks... maybe a better error handling on our could on each check that if it fails to check it's just not running?

Reply:
once we add an external network to a container in Docker Compose, every docker-compose command that parses the YML will fail case that network does not exist.

Therefore having an external network requires that every time we want run docker-compose we need to make sure that network exists.

Most commands are "OK"  to fail, as in, we don't need to go creating networks before each usage of DockerCompose, but in this case the `status` command is one we know can be used with containers running or not running, therefore I believe it makes sense we handle the network so we can properly run commands even though the containers are not up.

Reply:
By the way: having an external network é a feature I am adding due to the need for handling multi-projects in a next PR.